setup.py

```diff
@@ -1,109 +0,0 @@
-#!/usr/bin/env python
-
-import os
-import sys
-import glob
-import setuptools  # noqa: magic
-import numpy.f2py
-from numpy.distutils.core import setup, Extension
-from numpy.distutils.command.build_ext import build_ext
-from distutils.file_util import copy_file
-
-
-if not os.path.exists("src/fsps/libfsps/src/sps_vars.f90"):
-    raise RuntimeError(
-        "It looks like FSPS is not included in your source distribution. "
-        "If you're installing using git, don't forget to include the "
-        "submodules using a recursive clone or by running: \n"
-        "git submodule init\n"
-        "git submodule update"
-    )
```

Comment:
SciPy migrated this sort of logic to meson.build with the following example code:

```meson
fs = import('fs')
if not fs.exists('highs/README.md')
  error('Missing the `highs` submodule! Run `git submodule update --init` to fix this.')
endif
```

It might be useful to include the same sort of thing around line 8 of `src/fsps/meson.build`.

Reply:
@eli-schwartz — Thanks for this tips! I will incorporate them.